### DISCUSSION TITLE
read out AirPods

### DISCUSSION THREAD

**Discussion by inst5** [cables-gl/cables_docs#417] [Answers & Questions] [answered]
Hi,
I tried out the preview here: https://preview.undev.studio/temp/motion/ and this is great to use with mobile.

Is there a way to read the motion sensor of AirPods ?
I want to track head movements this way.

thanks for feedback.

**Comment by pandrr** [comment, marked as answer]
don't think thats possible in the browser

**Reply by inst5** [reply]
As that´s a pity.

In the meantime I found this: https://github.com/Mach1Studios/Pod-Mach1SpatialAPI
I contacted the devs to find out, if this works without an app
